### PR TITLE
Android demo

### PR DESCRIPTION
**Pull request by gjyoung1974**
This PR is to request an android demo.

This is a demo of an "Add payment card" form. 
1. The form includes Card.io to allow OCR input of a card
2. Mobile Backend API Client which persists token securely via VGS Proxy

@jviveiros @mjallday 
How about an Android demo?

Cheers!
~G

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.goyoung.android.vgs_cardform_demo"
```
Comment (mjallday) [inline android/app/build.gradle]: want a vgs namespace.

File: android/app/src/androidTest/java/com/vgs/android/vgs_cardform_demo/ExampleInstrumentedTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
```
Comment (mjallday) [inline android/app/src/androidTest/java/com/vgs/android/vgs_cardform_demo/ExampleInstrumentedTest.java]: don't need this

File: android/app/src/androidTest/java/com/vgs/android/vgs_cardform_demo/ExampleInstrumentedTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package com.vgs.android.vgs_cardform_demo;
```
Comment (mjallday) [inline android/app/src/androidTest/java/com/vgs/android/vgs_cardform_demo/ExampleInstrumentedTest.java]: io.vgs

File: android/app/src/main/java/com/vgs/android/vgs_cardform_demo/VGS_CardFormActivity.java [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    }
+
+    @Override
+    public void onCardFormSubmit() {
+
+
+        URL url = null;
+        try {
+            //todo: get this property from another means
+            url = new URL("https://tntlvnzzqsz.SANDBOX.verygoodproxy.com");
```
Comment (mjallday) [inline android/app/src/main/java/com/vgs/android/vgs_cardform_demo/VGS_CardFormActivity.java]: environment variable like in other examples https://github.com/verygoodsecurity/vgs-proxy-examples/blob/master/java/src/test/java/com/verygoodsecurity/example/ExampleTest.java#L31

File: node-payment-form/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# pay_form 
+Demo of Web-tier integration with the awesome data security capability at:
+https://www.verygoodsecurity.com/
+
+Demo at:
+[http://www.gordonyoung.us](http://www.gordonyoung.us)
```
Comment (mjallday) [inline node-payment-form/README.md]: doesn't respond for me!

**Comment by gjyoung1974** [comment]
going to refactor and make a new PR

**Pull request closed without merging**